tests/containers/bci_prepare.pm

```diff
@@ -75,13 +79,13 @@ sub run {
         }
         assert_script_run('pip3 --quiet install --upgrade pip', timeout => 600);
         assert_script_run("pip3 --quiet install tox", timeout => 600);
-    } elsif ($host_distri eq 'centos') {
+    } elsif ($host_distri eq 'centos' || $host_distri eq 'rhel') {
         foreach my $pkg (@packages) {
             assert_script_run("yum install -y $pkg", timeout => 300);
         }
         assert_script_run('pip3 --quiet install --upgrade pip', timeout => 600);
         assert_script_run("pip3 --quiet install tox", timeout => 600);
-    } elsif ($host_distri eq 'sles') {
+    } elsif ($host_distri eq 'sles' || $host_distri eq 'opensuse-leap') {
```

Comment:
```suggestion
    } elsif ($host_distri =~ m/sles|opensuse-leap/) {
```

Reply:
In the sles branch we have also the SUSEConnect handlers, so instead of doing another nested branches, I have decided to put it separated. But sure we can do it this way too.

Reply:
On the second thought, if we do `$host_distri =~ m/opensuse/` we could be possibly easily add Tumbleweed later on.

Reply:
Yes, let's just use `opensuse`!

Reply:
Alright, WDYT about the latest proposal?